### PR TITLE
Git hubwork flows build

### PR DESCRIPTION
**Pull request by chandantiwari1**
add build in the workflows

### CODE REVIEW

File: apps/web/package.json [hunk 1]
```diff
@@ -9,13 +9,13 @@
     "lint": "next lint"
   },
   "dependencies": {
-    "@meway/api": "*",
-    "@meway/tailwind-config": "*",
+    "@meway/api": "file:../../packages/api",
```
Comment (sutyum) [inline apps/web/package.json:12]: why is this needed?

File: apps/web/package.json [hunk 2]
```diff
@@ -9,13 +9,13 @@
...
+    "@meway/tailwind-config": "file:../../packages/config/tailwind",
     "next": "13.0.0",
-    "react": "18.2.0",
-    "react-dom": "18.2.0",
+    "react": "18.0.0",
+    "react-dom": "18.0.0",
     "eslint-config-custom": "*",
-    "react-native": "0.70.6"
+    "react-native": "0.69.6"
```
Comment (sutyum) [inline apps/web/package.json:18]: why is this needed?

File: apps/web/package.json [hunk 3]
```diff
@@ -9,13 +9,13 @@
...
   },
   "dependencies": {
-    "@meway/api": "*",
-    "@meway/tailwind-config": "*",
+    "@meway/api": "file:../../packages/api",
+    "@meway/tailwind-config": "file:../../packages/config/tailwind",
     "next": "13.0.0",
-    "react": "18.2.0",
-    "react-dom": "18.2.0",
+    "react": "18.0.0",
```
Comment (sutyum) [inline apps/web/package.json:15]: why is this needed?

File: apps/meway-tablet/src/_app.tsx [hunk 4]
```diff
@@ -1,6 +1,7 @@
 import { StatusBar } from "expo-status-bar";
 import React from "react";
 import { SafeAreaProvider } from "react-native-safe-area-context";
+import "react-native-get-random-values";
```
Comment (sutyum) [inline apps/meway-tablet/src/_app.tsx:4]: why is this needed?

File: apps/meway-tablet/package.json [hunk 5]
```diff
@@ -4,42 +4,45 @@
...
+    "dev": "expo start",
     "clean": "rm -rf .expo .turbo node_modules",
-    "web": "expo start --web",
-    "postinstall": "cd ../.. && pnpm build"
+    "web": "expo start --web"
   },
   "dependencies": {
-    "@meway/api": "*",
-    "@meway/tailwind-config": "*",
+    "@meway/api": "file:../../packages/api",
```
Comment (sutyum) [inline apps/meway-tablet/package.json:12]: why?

File: apps/meway-tablet/package.json [hunk 6]
```diff
@@ -4,42 +4,45 @@
   "main": "index.ts",
   "scripts": {
     "start": "expo start",
-    "dev": "expo start --android",
```
Comment (sutyum) [inline apps/meway-tablet/package.json:7]: why?

**Review by sutyum — CHANGES_REQUESTED** [review]

File: apps/web/package.json (hunk 1 above)
Comment (chandantiwari1) [reply apps/web/package.json]: sir yarn workspace error, so i converted pnpm packages to the yarn

File: apps/web/package.json (hunk 2 above)
Comment (chandantiwari1) [reply apps/web/package.json:17]: react-native peer dependencies error so that I deprecetated it to 9.69.6

File: apps/web/package.json (hunk 3 above)
Comment (chandantiwari1) [reply apps/web/package.json:14]: peer dependencies error

File: apps/meway-tablet/src/_app.tsx (hunk 4 above)
Comment (chandantiwari1) [reply apps/meway-tablet/src/_app.tsx:4]: to uuid random values error

File: apps/meway-tablet/package.json (hunk 5 above)
Comment (chandantiwari1) [reply apps/meway-tablet/package.json]: yarn workspaces error

File: apps/meway-tablet/package.json (hunk 6 above)
Comment (chandantiwari1) [reply apps/meway-tablet/package.json:7]: I will fix this in the next commit.

**Pull request closed without merging**